src/UXClient/Components/Marker/Marker.ts

```diff
@@ -507,10 +507,12 @@ class Marker extends Component {
     }
 
     public focusCloseButton () {
-        this.closeButton.node().focus();
+        if (this.closeButton) {
+            this.closeButton.node().focus();
```

Comment:
Could use optional chaining.. this sweeeeet new feature
```suggestion
            this.closeButton?.node().focus();
```

Reply:
Totally up to you though, ofc